src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -397,4 +400,18 @@ protected function getTemplateManager()
 
         return $this->templateManager;
     }
+
+    private function renderWithCspNonces(Request $request, $template, $variables, $code = 200, $headers = array('Content-Type' => 'text/html'))
```

Comment:
Actually this PR provides CSP support for setups where `unsafe-inline` is disabled in **development** environment (because it only affects the WebProfilerBundle that should not be enabled in prod in any case)

So, even if it's recommended to use signature instead of nonce, it's a recommendation for production or staging. In development, it enables user to fully uses test with their CSP and uses the WebProfilerBundle.

So, no, it's not a problem :)